From 92f732e60515bb0261c4b9565b33107bf7ab0471 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Wed, 4 Feb 2015 16:18:26 +0800
Subject: [PATCH 0096/5242] MLK-10202-1 ARM: imx6sx-sdb: add tx-d-cal property
 for usbphy

commit  39876f57b1a6e37ecfbb30d68464d2eaffd063d4 from
https://source.codeaurora.org/external/imx/linux-imx.git

It is used to improve signal quality for board.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dts |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 6dd9beb..9447f7b 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -136,3 +136,11 @@
 &reg_soc {
 	vin-supply = <&sw1a_reg>;
 };
+
+&usbphy1 {
+	tx-d-cal = <0x5>;
+};
+
+&usbphy2 {
+	tx-d-cal = <0x5>;
+};
-- 
1.7.9.5

